include/ginkgo/core/log/record.hpp

```diff
@@ -224,6 +237,7 @@ struct criterion_data {
  * events, all parameters are cloned. If it is sufficient to clone one
  * parameter, consider implementing a specific logger for this. In addition, it
  * is advised to tune the history size in order to control memory overhead.
+ *
```

Comment:
Why all these added empty comment lines? I would guess it was due to replacing stuff through a script.